server/plugins/renderer/responses/pencil-response.js

```diff
@@ -93,10 +93,10 @@ const makeDecorator = (request, context) => content => {
 
 module.exports = function (data, assembler) {
     this.respond = function (request, h) {
-        const templateEngine =  data.context.template_engine || "handlebars-v3";
+        const templateEngine =  data.context.template_engine || "handlebars_v3";
 
         if (!isSupportedHandlebarsVersion(templateEngine)) {
-            throw new Error('Provided Handlebars version is not supported! Please use:handlebars-v3, handlebars-v4');
+            throw new Error('Provided Handlebars version is not supported! Please use:handlebars_v3, handlebars_v4');
         }
 
         const paper = new Paper(data.context.settings, data.context.theme_settings, assembler, templateEngine);
```

Comment:
I think we should change `handlebars_v3/4` to `handlebars-v3/4` before we pass it down to paper. So we don't have to make changes to paper.

Reply:
Have you checked my comment on paper? https://github.com/bigcommerce/paper/pull/213#issuecomment-693985620